src/components/Work/Work.tsx

```diff
@@ -69,19 +69,23 @@ const WorkDetail: React.FC<{ workResult: WorkResult }> = (props) => {
             </div>
           )}
         </div>
-
         <div>
           <DS.Heading level={2} id="featured-edition">
             Featured Edition
           </DS.Heading>
         </div>
-
-        <div>
-          <EditionCard
-            edition={featuredEdition}
-            title={work.title}
-          ></EditionCard>
-        </div>
+        {featuredEdition ? (
```

Comment:
So currently in DRB what happens when a work has no editions is that only the data table shows up:  https://digital-research-books-beta.nypl.org/edition/7425982?showAll=false

I think there is a separate ticket here to create a good fallback state for a work with no editions, but I think it makes sense to mimic this functionality for now, such that we can create that fallback once and for all.

Reply:
The Edition page handles this by using the first available instance instead of requiring the link to be not be catalog, so this does not occur on that page.